nengo/spa/pointer.py

```diff
@@ -76,14 +80,15 @@ def __isub__(self, other):
     def __mul__(self, other):
         """Multiplication of two SemanticPointers is circular convolution.
 
-        If mutliplied by a scaler, we do normal multiplication.
+        If mutliplied by a scalar, we do normal multiplication.
```

Comment:
What do you call it when two dogs of different breeds have babies? Mutliplication!